fix hardcoded MPI-related variables for EB-provided Intel MPI
author: Kenneth Hoste (Ghent University)
--- PELICANS/etc/extra-Linux.mak.orig	2015-09-04 11:28:14.263778000 +0200
+++ PELICANS/etc/extra-Linux.mak	2015-09-04 11:31:37.180400000 +0200
@@ -152,10 +152,10 @@
 CPPFLAGS += -I$(PELICANSHOME)/$(EXTPACKS)/MPI/include -DMPIRUN=\"$(MPIRUN)\"
 endif
 
-CPPFLAGS += -I$(MPIPATH)/include
-MPIPATH   = $(EXTRA_LIBS_DIR)/openmpi
-LIBPATH  += $(MPIPATH)/lib
-LDLIBS   += -lmpi -lmpi_cxx
+MPIPATH   = $(EBROOTIMPI)
+CPPFLAGS += -I$(MPIPATH)/include64
+LIBPATH  += $(MPIPATH)/lib64
+LDLIBS   += -lmpi -lmpicxx
 endif
 
 
